build.gradle

```diff
@@ -6,19 +6,6 @@ enum VersionType {
     SNAPSHOT
 }
 
-ext {
-    graalRevision = getGraalRevision()
-    project.logger.info("Graal revision: $graalRevision")
-
-    if (version.endsWith("-SNAPSHOT")) {
-        versionType = VersionType.SNAPSHOT
-    } else if (version.matches(/.+\-RC[0-9]+$/)) {
-        versionType = VersionType.RELEASE_CANDIDATE
-    } else {
-        versionType = VersionType.GA_RELEASE
-    }
-}
-
 allprojects {
```

Comment:
I wonder if the problem is this allprojects block. I'm looking at the core sdk repo and we don't use it. For example, the `ext` block is defined outside in the main body on the build.gradle.

Reply:
Giving a try to that, have a look.

Reply:
I.e. I removed `allprojects`

Reply:
Let's wait for the build though